src/app/Caches/SetAvailableCache.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Button, Modal, Text, TextContent } from '@patternfly/react-core';
+import { useApiAlert } from '@app/utils/useApiAlert';
+import { useCaches } from '@app/services/dataContainerHooks';
+import { useTranslation } from 'react-i18next';
+import {ConsoleServices} from "@services/ConsoleServices";
+import {CheckCircleIcon} from "@patternfly/react-icons";
+
+/**
+ * Set Available cache modal
+ */
+const SetAvailableCache = (props: {
+  cacheName: string;
+  isModalOpen: boolean;
+  closeModal: (boolean) => void;
+}) => {
+  const { addAlert } = useApiAlert();
+  const { reloadCaches } = useCaches();
+
+  const { t } = useTranslation();
+  const brandname = t('brandname.brandname');
+
+  const clearSetAvailableCacheModal = (ignoreDone: boolean) => {
+    props.closeModal(ignoreDone);
+  };
+
+  const handleAvailableButton = () => {
+    if (props.cacheName) {
+      ConsoleServices.caches()
+        .setAvailability(props.cacheName)
+        .then((actionResponse) => {
+          clearSetAvailableCacheModal(actionResponse.success);
+          addAlert(actionResponse);
+          reloadCaches();
+        });
+    }
+  };
+
+  return (
+    <Modal
+      titleIconVariant={CheckCircleIcon}
+      className="pf-m-redhat-font"
+      width={'50%'}
+      isOpen={props.isModalOpen}
+      title="Make the cache Available"
```

Comment:
Extract these into the properties file.

Ask @oraNod for rewording please :)))